app/policies/article_policy.rb

```diff
@@ -1,29 +1,17 @@
 class ArticlePolicy < ApplicationPolicy
   def show?
-    manage? || @record.status == "published"
-  end
-
-  def new?
-    user.present?
+    record.status == "published" || update?
   end
 
   def create?
     user.present?
```

Comment:
I think you can use `authenticate_user!` method from devise in controllers instead of using policy for checking existing current user (https://github.com/heartcombo/devise#controller-filters-and-helpers)